index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a>My Logo</a>
+        </div>
+        <nav class="nav-bar">
+            <div class = "ham-menu">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg1">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg2">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg3">
+           </div>
+        </nav>
+    </header>
+    <main>
+       <section>
+           <div class="Backgroud">
+  
+    
+           </div>
+        <div class="title-name">
+            <h2>I’m Amol
+                Glad to see you!
+            </h2>
+        </div>
+        <div class="title-intro">
+            <p>I’m a software developer! I can help you build a product , feature or website
+                 Look through some of my work and experience! If you like what you see 
+                 and have a project you need coded, don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="Connect" >
+            <a class="dontwrap">LET'S CONNECT</a>
+        </div>
+        <div >
+       <img src="image/SocialIcons.png" alt="Socail Media Icons" class="SocialIcon">
+        </div>
```

Comment:
- [ ] As requested by the previous reviewer, kindly add the list of icons individually, instead of adding all as a single image.

Reply:
Implemented 👍